Gordon360/Models/CCT/InvolvementOffering.cs

```diff
@@ -0,0 +1,32 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT
+{
+    [Keyless]
+    public partial class InvolvementOffering
+    {
+        [Required]
+        [StringLength(8)]
+        [Unicode(false)]
+        public string ActivityCode { get; set; }
+        [StringLength(45)]
+        [Unicode(false)]
+        public string ActivityDescription { get; set; }
+        [StringLength(3)]
+        [Unicode(false)]
+        public string ActivityType { get; set; }
+        [StringLength(60)]
+        [Unicode(false)]
+        public string ActivityTypeDescription { get; set; }
+        [Required]
+        [StringLength(8)]
+        [Unicode(false)]
+        public string SessionCode { get; set; }
+    }
+}
```

Comment:
Same thing here. This is a part of another one of my PRs, so you should not be including it in your unrelated work.